osm2lanes-web/src/control.rs

```diff
@@ -175,4 +254,14 @@ impl Component for Control {
             </>
         }
     }
+
+    fn rendered(&mut self, ctx: &Context<Self>, first_render: bool) {
+        if first_render {
+            let handle = {
+                let link = ctx.link().clone();
+                Timeout::new(1, move || link.send_message(Msg::FirstLazy))
+            };
+            handle.forget();
+        }
```

Comment:
hacky...

Reply:
Why don't we do all the work of loading examples up-front and avoid the indirection? Is it slow enough to block the page load or something?

Reply:
I noticed the chance, I think 50% of the page load is on deserializing the .yml